src/forms/selectInput.test.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import SelectInput from './SelectInput';
+
+const testOptions = [
+	{ label: 'One', value: '1' },
+	{ label: 'Two', value: '2' },
+	{ label: 'Three', value: '3' }
+];
+
+describe('SelectInput basic', () => {
+	let component;
+
+	beforeEach(() => {
+		component = TestUtils.renderIntoDocument(
+			<SelectInput
+				label='Test select'
+				name='testSelect'
+				options={testOptions}
+			/>
+		);
+	});
+	afterEach(() => {
+		component = null;
+	});
+
+	it('exists', () => {
+		expect(() => TestUtils.findRenderedDOMComponentWithTag(component, 'select')).not.toThrow();
+	});
+
+	it('should have a NAME attribute', () => {
+		const selectEl = TestUtils.findRenderedDOMComponentWithTag(component, 'select');
+		expect(selectEl.name).toEqual('testSelect');
```

Comment:
in the future, set any 'expected' string values to constants to enforce a connection between the input and output